docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+
+services:
+  app_dev:
+    build:
+      context: ./
+      dockerfile: DockerfileDev
+    ports:
+      - "8001:8001"
+    volumes:
+      - ./:/app
```

Comment:
the volumes ensure that container and host share these files. 
Meaning any change made on the host will be immediately reflected within the container as well.

Reply:
Ok, this looks good.